Swiftfin/Components/LearnMore/Components/ItemDescription.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+
+struct ItemDescription: Identifiable {
```

Comment:
Instead of making another type, just use a `TextPair`. In the future, we may remove this and instead move to `LabeledContent` from SwiftUI.

Reply:
Sounds good. I've make this change but I just wanted to confirm UI wise we liked how this looked:

![Simulator Screenshot - iPhone 16 Pro - 2024-11-14 at 21 41 50](https://github.com/user-attachments/assets/484aa8ca-c15a-436d-8b48-5d10e9b8a4a5)

Reply:
I don't have a different design in mind so that's good 👍